src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -294,6 +299,33 @@ protected function bootstrappers()
     }
 
     /**
+     * Set environment if 'env' option if provided.
+     *
+     * @param  InputInterface  $input
+     * @return void
+     */
+    protected function setEnvironment($input)
+    {
+        if (! app()->runningInConsole()) {
```

Comment:
@GrahamCampbell If i write this line:

    $this->app->call(''Illuminate\Foundation\Application@method');

I can access the methods i mentioned without using app global helper.